package.json

```diff
@@ -100,6 +100,7 @@
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-react": "^7.20.0",
     "eslint-plugin-react-hooks": "^4.0.4",
+    "glob": "^7.1.6",
```

Comment:
Thanks. I was sure I'd add this already. Spent hours thinking it wanted me to move the dependency into `dependencies` and trying to fix that.. ugh

Reply:
:) maybe you didn't use the -W option to add dev dependencies in monorepo root ;)